File: civitasJAVA/civitas/src/civitas/testP3.java
package civitas;

import java.util.Arrays;
import java.util.ArrayList;

import JuegoTexto.Controlador;
import JuegoTexto.VistaTextual;


public class testP3 {
    
     public static void main(String[] args) {
        CivitasJuego juego = new CivitasJuego(new ArrayList<>(Arrays.asList("Pepe","Juan","Marta")));
        VistaTextual vista = new VistaTextual();
        Controlador controlador = new Controlador(juego,vista);
        
        Dado.getInstance().setDebug(true);
        controlador.juega();
     }
}

File: civitasRUBY/lib/test_p3.rb
#encoding: utf-8

require_relative "civitas_juego"
require_relative "vista_textual"
require_relative "dado"

module Civitas
  class TestP3
    
    def self.main
      
      juego=CivitasJuego.new( ["Pepe","Juan","Marta"] )
      vista = VistaTextual.new
      controlador= Controlador.new(juego,vista)
      Dado.instance.debug(true)
      
      controlador.juega()     
    end
  end
  Civitas::TestP3.main
end
